crates/wdk-build/rust-driver-makefile.toml

```diff
@@ -177,13 +180,16 @@ args = [
   "1.33",
 ]
 
+
 [tasks.infverif]
 private = true
 dependencies = ["stampinf"]
+condition = { env_not_set = ["WDK_INF_NEW_VERSION"] }
```

Comment:
this should be set only in samples (ie. WindowsRustDriverSamples repo + sample_kmdf_driver makefile). you could do something like add the following snippets to their local makefiles:

```
[tasks.infverif]
condition = { env_not_set = ["WDK_INF_NEW_VERSION"] }
```

This will only overried the `condition` for `tasks.infverif` locally for flows run in those dirs

Reply:
Thanks for the pointer, changing it.